docs/UserGuide.md

```diff
@@ -57,8 +57,8 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+* Parameters that is expected to only appear once in the command will be treated as an invalid command if you specified it multiple times.<br>
+  e.g. if you specify `p/12341234 p/56785678`, the application will treat this as an invalid command.
```

Comment:
Good catch.
1. As this was a 'special' behavior, we had to specifically mention it in the UG. But now that we have removed that special behavior, just removing the said documentation is enough. No need to explain the updated behavior, as that is what one would expect anyway.
2. This UG change is tightly related to the behavior change. Should be in the same commit. Each commit should take the code from one 'correct' state to another 'correct' state. A commit that updates the behavior without updating the UG takes the code to an incorrect state.